@xen-orchestra/lite/src/components/pif/PifRow.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <tr class="pif-row" @click="redirect()">
+    <td v-tooltip class="typo p3-regular text-ellipsis host-container">
+      <div v-if="host" class="host">
+        <UiObjectIcon :state="hostPowerState" type="host" size="small" />
+        <span v-tooltip class="typo p3-regular text-ellipsis host-name">
+          {{ host.name_label }}
+        </span>
+      </div>
+      <div v-else>
+        <span>{{ $t('host-unknown') }}</span>
+      </div>
+    </td>
+    <td v-tooltip class="typo p3-regular text-ellipsis device">{{ pif.device }}</td>
+    <td v-tooltip class="typo p3-regular status">
+      <VtsConnectionStatus v-if="status !== undefined" :status />
+    </td>
+    <td>
+      <UiButtonIcon size="small" accent="brand" :icon="faAngleRight" />
+    </td>
+  </tr>
+</template>
+
+<script setup lang="ts">
+import type { XenApiPif } from '@/libs/xen-api/xen-api.types'
+import { useHostMetricsStore } from '@/stores/xen-api/host-metrics.store'
+import { useHostStore } from '@/stores/xen-api/host.store'
+import { usePifStore } from '@/stores/xen-api/pif.store'
+import VtsConnectionStatus, { type ConnectionStatus } from '@core/components/connection-status/VtsConnectionStatus.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiObjectIcon from '@core/components/ui/object-icon/UiObjectIcon.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+import { useRouter } from 'vue-router'
+
+const { pif } = defineProps<{
+  pif: XenApiPif
+}>()
+
+const { getByOpaqueRef } = useHostStore().subscribe()
+const { isHostRunning } = useHostMetricsStore().subscribe()
+const { getPifStatus } = usePifStore().subscribe()
+
+const router = useRouter()
+
+const status = computed<ConnectionStatus | undefined>(() => getPifStatus(pif))
```

Comment:
1. This should not be typed here
2. getPifStatus always returns a `string`, never `undefined`

```suggestion
const status = computed(() => getPifStatus(pif))
```

Reply:
Yes, little oversight